spec/lib/configuration_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+module FloodRiskEngine
+  describe Configuration do
+    it "adds #config to the engine module" do
+      expect(FloodRiskEngine).to respond_to(:config)
+    end
+
+    it "has the correct default values" do
+      expect(FloodRiskEngine.config.test_me).to eq "test"
+    end
```

Comment:
I'd remove this test as the 'is configurable via a block' test tests the functionality without relying on the presence of special - only for testing - configuration item.